Dockerfile

```diff
@@ -532,6 +532,22 @@ RUN \
 
 # =^..^=   =^..^=   =^..^=   =^..^=   =^..^=   =^..^=   =^..^=   =^..^=   =^..^=
 
+FROM sdk-rust as sdk-license-tool
+
+USER root
+COPY license-tool /home/builder/license-tool
+RUN \
+  mkdir -p /usr/libexec/tools /home/builder/license-tool /usr/share/licenses/bottlerocket-license-tool && \
+  chown -R builder:builder /usr/libexec/tools /home/builder/license-tool /usr/share/licenses/bottlerocket-license-tool
```

Comment:
It seems a little odd to COPY to a location and then immediately `mkdir -p` the same location. I suspect the COPY needs to come after `WORKDIR /home/builder/license-tool`